app/vili.go

```diff
@@ -256,6 +257,14 @@ func New() *App {
 				DeployUsernames: util.NewStringSet(config.GetStringSlice(config.SlackDeployUsernames)),
 			})
 		},
+		// set up circleci service
```

Comment:
What happens when a user of the open source vili tool does not provide CircleCI auth tokens?  Will this block and fail startup?